docs/versions/netstandard2.0_diff.md

```diff
@@ -827,33 +1222,102 @@
 +        public virtual Assembly EndInvoke(IAsyncResult result);
 +        public virtual Assembly Invoke(object sender, ResolveEventArgs args);
 +    }
++    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
++    public struct RuntimeArgumentHandle
```

Comment:
this is still missing `{ }`

Reply:
Those ones I don't care about right now. Fixing this would probably require some crazy gymnastics which I don't have time for right now :-)